app/src/main/java/org/kiwix/kiwixmobile/localFileTransfer/WifiDirectManager.kt

```diff
@@ -133,10 +134,7 @@ class WifiDirectManager @Inject constructor(
       }
 
       override fun onFailure(reason: Int) {
-        Log.d(
-          TAG, "${activity.getString(R.string.discovery_failed)}: " +
-            getErrorMessage(reason)
-        )
+        Log.d(TAG, "${activity.getString(R.string.discovery_failed)}: " + getErrorMessage(reason))
```

Comment:
can use string interpolation instead of concatenation

Reply:
done